### PR TITLE
Enable opentracing for tokeninfo

### PR DESCRIPTION
**Pull request by nuwandiw**
For https://github.com/zalando-stups/taupage/issues/566

### CODE REVIEW

File: runtime/etc/init/planb-tokeninfo.conf [hunk 1]
```diff
@@ -18,6 +18,11 @@ script
             export ENABLE_INTROSPECTION="true"
         fi
         if [ -n "$config_local_planb_tokeninfo_lightstep_token" ]; then
+            if [[ $config_local_planb_tokeninfo_lightstep_token == "aws:kms:"* ]]
+            then
+              config_local_planb_tokeninfo_lightstep_token=${config_local_planb_tokeninfo_lightstep_token##aws:kms:}
+              config_local_planb_tokeninfo_lightstep_token=`python3 /opt/taupage/bin/decrypt-kms.py $config_local_planb_tokeninfo_lightstep_token`
```
Comment (aermakov-zalando) [inline runtime/etc/init/planb-tokeninfo.conf]: Do you mind running it via `shellcheck` and fixing the issues? You can obviously ignore the warnings about undefined variables, but the rest is fixable.

Comment (nuwandiw) [reply runtime/etc/init/planb-tokeninfo.conf]: I ran the shellcheck and did some changes

**Comment by aermakov-zalando** [comment]
Thanks!

**Comment by aermakov-zalando** [comment]
👍

**Comment by jrake-revelant** [comment]
👍

**Pull request merged**